modules/nf-core/mapad/map/meta.yml

```diff
@@ -0,0 +1,81 @@
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/yaml-schema.json
+name: "mapad_map"
+description: Map short-reads to an indexed reference genome
+keywords:
+  - mapad
+  - short-read
+  - align
+  - reference
+  - fasta
+  - bam
+  - map
+  - fastq
+tools:
+  - "mapad":
+      description: "An aDNA aware short-read mapper"
+      homepage: "https://github.com/mpieva/mapAD"
+      documentation: "https://github.com/mpieva/mapAD"
+      tool_dev_url: "https://github.com/mpieva/mapAD"
+      licence: "['MIT']"
+
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+  - reads:
+      type: file
+      description: BAM/CRAM/FASTQ/FASTQ.GZ file
```

Comment:
Unmapped BAM? Or does it allow remapping?  I would put the description more like 'Sequencing reads in BAM or FASTQ related formats' and leave the actual suffixes to the pattern. Would `fq.gz` and `fq` also be supported?

Reply:
Should also specify it's a mandatory file input